app/code/Magento/Catalog/Test/Mftf/Test/StorefrontPurchaseProductCustomOptionsDifferentStoreViewsTest.xml

```diff
@@ -48,6 +48,8 @@
         </before>
 
         <after>
+            <!-- Customer Log Out -->
+            <actionGroup ref="StorefrontCustomerLogoutActionGroup" stepKey="customerLogout"/>
```

Comment:
Please use the same stepkey: customerLogoutStorefront

Reply:
updated.